inlang/source-code/website/src/services/auth/src/onSignOut.ts

```diff
@@ -1,25 +1,13 @@
 import type { SetStoreFunction } from "solid-js/store"
 import type { LocalStorageSchema } from "../../../services/local-storage/index.js"
-import { publicEnv } from "@inlang/env-variables"
 import { telemetryBrowser } from "@inlang/telemetry"
 import { browserAuth } from "@lix-js/client"
 
 /**
  * This function is called when the user clicks the "Sign Out" button.
  */
 export async function onSignOut(args: { setLocalStorage: SetStoreFunction<LocalStorageSchema> }) {
-	// sign out on the server
-	const hasExternalGitProxy =
-		publicEnv.PUBLIC_SERVER_BASE_URL !== publicEnv.PUBLIC_GIT_PROXY_BASE_URL
-
-	if (hasExternalGitProxy) {
-		await Promise.allSettled([
-			fetch("/services/auth/sign-out", { method: "POST" }),
-			browserAuth.logout(),
-		])
-	} else {
-		await fetch("/services/auth/sign-out", { method: "POST" })
-	}
+	await browserAuth.logout()
```

Comment:
de-coupling auth from the `repo` object seems to be the right design decision :)